Package.swift

```diff
@@ -1,5 +1,4 @@
-// swift-tools-version:5.4
-// The swift-tools-version declares the minimum version of Swift required to build this package.
+// swift-tools-version:5.5
```

Comment:
Is this the correct version?

Reply:
> Is this the correct version?

may be not, it was for DocC, but i think its will be ignored on unsupported version.